cypress/fixtures/listWorkflowsFlow.json

```diff
@@ -1 +1,195 @@
\ No newline at end of file
+{
+    "blocks": [
+        {
+            "type": "init",
+            "blockComment": "Mocked"
```

Comment:
Added the mocked comment, for sanity checking

Reply:
Thank you for formatting this file and for commenting what you have changed! :pray: